www/source/partials/docs/_dev-pkgs-plan-builds.html.md.erb

```diff
@@ -57,6 +57,19 @@ Depending on the platform of your host and your Docker configuration, the behavi
 
 > Note: For more details related to Windows containers see [Running Habitat Windows Containers](/docs/best-practices/#running-habitat-windows-containers).
 
+### Building dependent plans in the studio
```

Comment:
We should capitalize the S in Studio.

Reply:
Prefer title-case: "Building Dependent Plans in the Studio"